### ISSUE TITLE
Support for Cloudflare Worker stats

### ISSUE THREAD

**Issue by dmyers** [wehkamp/docker-prometheus-cloudflare-exporter#18]
Do you think this would be a useful feature? I'm looking to get my stats for my Cloudflare worker somehow into Prometheus and curious if this package could do it possibly with a PR or something.

https://developers.cloudflare.com/analytics/graphql-api/tutorials/querying-workers-metrics
https://developers.cloudflare.com/workers/about/metrics/#invocation-statuses

**Comment by harshal-shah** [comment]
Yes, IMO workers status could be a nice to have feature in this exporter
